appengine-python-flask/main.py

```diff
@@ -175,7 +177,7 @@ def shout():
     entity.host = socket.gethostname()
     async_put = entity.put_async()
 
-    # Insert the task into the work queue.
+    # Publish a shout request to the Pub/Sub topic.
```

Comment:
I think it would be clearer to phrase in terms of publishing messages throughout, which would let them map these concepts to the pubsub docs.
E.g. along the lines of  "Publish a 'shout request' message"